### DISCUSSION TITLE
Show borders of national park

### DISCUSSION THREAD

**Discussion by stonecrusher** [osmandapp/OsmAnd#11745] [Q&A] [answered]
OSM has this nice view of areas:
![2021-05-23 20_12_48-Relation_ ‪Nationalpark Sächsische Schweiz‬ (‪1595534‬) _ OpenStreetMap](https://user-images.githubusercontent.com/1388389/119271791-53b18a00-bc03-11eb-97c1-cf0de0947790.png)

Now if I am outside with OsmAnd and want to determine if I am still in the area of the national park or not, how can I do that?

**Comment by vshcherb** [comment, marked as answer]
I don't think it's possible today.

**Reply by stonecrusher** [reply]
That explains why I couldn't find it ;-) Thank you for your answer!
As special rules apply to national parks, it's important to know if you're still in or outside. I'll find a way.

**Reply by pebogufi** [reply]
@stonecrusher 
This is a example to find and export regional borders, i suppose, with a similar script it might be possible to create gps tracks with borders of national parks.
When showing this track you see theese borders on screen.

https://overpass-turbo.eu/#

  area[name="Landkreis Breisgau-Hochschwarzwald"][admin_level=6][boundary=administrative]->.lk_breisgau_hochschwarzwald ;
  rel(pivot.lk_breisgau_hochschwarzwald);
  out geom;

**Reply by stonecrusher** [reply]
Thank you for the hint.
I already started reading the docs, but syntax is not clear to me yet. I hope I'll figure it out. It seems to have an export function!

**Reply by thompehw** [reply]
@pebogufi 's code is a helpful example of how to use "pivot" in the full Overpass Query Language (thanks!). But it can be done more easily in Overpass Turbo using the Wizard. Display the area of interest, open the wizard and type something like:

boundary=national_park or (boundary=protected_area and designation=national_park)

build and run, play with it until you're happy, then export as GPX.

**Reply by stonecrusher** [reply]
@thompehw Your proposal already works perfect for me and much better than the stuff I typed into the wizard. National park borders and core zone borders in one export file for offline use! I'm happy, thank you!
And as always, the solution was much easier than I thought. Didn't even know what a bbox is but it turns out I don't have to.

**Comment by maltfield** [comment]
Is there a feature request open for this yet?

**Comment by maltfield** [comment]
This is possible today. Open the settings menu and go to:

Configure Map -> Details -> Nature reserve

Make sure it's enabled. This should display the borders of both [nature reserves](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dnature_reserve) and [national parks](https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dnational_park).

**Reply by tommycrock** [reply]
But I don't see borders of UK national parks, which have protect class=5
